docs/authentication.md

```diff
@@ -0,0 +1,47 @@
+# Authentication

```

Comment:
This and the other page should be included in our MkDocs config: https://github.com/PrairieLearn/PrairieLearn/blob/master/mkdocs.yml. Now that we have several pages, I'd recommend introducing a new section, something like "Running in production", and placing these and the existing "Running in production" page under it.

Reply:
Add new section and moved docs